wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/ProxyCommand.java

```diff
@@ -21,7 +22,9 @@ public class ProxyCommand extends Command {
 
   /**
    * Creates a new ProxyCommand that schedules the supplied command when initialized, and ends when
-   * it is no longer scheduled. Useful for lazily creating commands at runtime.
+   * it is no longer scheduled. Useful for lazily creating proxied commands at runtime. Proxying
+   * should only be done if truly necessary, if only runtime command construction is needed, use
```

Comment:
suggestion: "proxying should only be done if there is a need to escape from the composition requirement semantics"